calico/felix/fetcd.py

```diff
@@ -334,6 +335,54 @@ def __init__(self, config, etcd_api, status_reporter, hosts_ipset):
         self.last_rate_log_time = monotonic_time()
         # Register for events when values change.
         self._register_paths()
+        self.estimated_host_count()
+        self._usage_count_greenlet = gevent.Greenlet(
+            self._periodically_usage_count
+        )
+        # self._usage_count_greenlet.link_exception(self._on_worker_died)
```

Comment:
This worked in the Etcd thread, but caused a crash in the Watcher thread.  It seems like we'd want this functionality, but I wasn't sure...